src/sentry/static/sentry/app/routes.jsx

```diff
@@ -1056,10 +1049,24 @@ function routes() {
         </Route>
 
         <Route path=":projectId/" component={errorHandler(ProjectDetails)}>
-          <IndexRoute component={errorHandler(Stream)} />
-          <Route path="issues/" component={errorHandler(Stream)} />
+          <IndexRoute
+            componentPromise={() =>
+              import(/*webpackChunkName: "Stream"*/ './views/stream')}
```

Comment:
```suggestion
              import(/* webpackChunkName: "Stream" */ './views/stream')}
```

Are the spaces important in these comments?

Reply:
I'm actually not 100% sure, adding anyway though